index.html

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script type="module">
+      import RefreshRuntime from "http://vite-project.ddev.site:5273/@react-refresh"
+      RefreshRuntime.injectIntoGlobalHook(window)
+      window.$RefreshReg$ = () => {}
+      window.$RefreshSig$ = () => (type) => type
+      window.__vite_plugin_react_preamble_installed__ = true
+    </script>
```

Comment:
Had to recreate how Vite was transforming the html file.

Note the `import` at the top is referencing Vite's dev server URL.

Reply:
Wow, would have never figured that out. 🤯